wom/src/main/scala/wom/graph/GraphNode.scala

```diff
@@ -80,6 +82,9 @@ object GraphNode {
     var _graphNode: A = _
     private def getGraphNode = _graphNode
     def get: Unit => A = _ => getGraphNode
+    def isAssignedAndOfType[T: ClassTag]: Boolean = {
+      Option(_graphNode) exists classTag[T].runtimeClass.isInstance
```

Comment:
FMI: Is this better than .isInstanceOf, or a simple match?

Reply:
It turns out to be required for an API that looks like this since otherwise the type is erased. I'll leave a comment taunting the reader to improve it if they can.